03_building_and_packaging/examples/cmake/main.cpp

```diff
@@ -2,6 +2,10 @@
 
 int main(int argc, char *argv[])
 {
+  if (argc > 1)
+  {
+   
+  }
```

Comment:
Where does this change come from? Like... why is the content of this file changed?